src/main/resources/db/migration/V1_1__create_user_table.sql

```diff
@@ -0,0 +1,2 @@
+alter table referral
+    add column created_by_user_id text;
```

Comment:
If we expect multiple auth sources, I think we'd need a way to signal to the consumers of our data the source of the identity. To somehow to mark that this `user_id` is "from delius", etc.

Random idea: we could use JSONB columns with `{"id": "12345", "auth_source": "delius"}` content. Or any other, more sensible options :)